pyproject.toml

```diff
@@ -42,7 +42,7 @@ asyncio_mode = "strict"
 name = "remote-settings"
 version = "0"
 description = "Easily manage evergreen settings data in Firefox"
-authors = ["@mozilla/remote-settings-reviewers"]
+authors = ["Contactless <postmaster@localhost>"]
```

Comment:
This is a frankly `very annoying` feature of Poetry.

I think this is happening because even though `make install-docs` specifies we're only installing the docs dependencies, it still tries to install the root project.

We could also make this error go away with this change:

```diff
diff --git a/Makefile b/Makefile
index cb5751b..b51a823 100644
--- a/Makefile
+++ b/Makefile
@@ -86,7 +86,7 @@ down:  ## Shutwdown all containers
 
 install-docs: $(VENV)/bin/python $(DOC_STAMP)  ## Install documentation build dependencies
 $(DOC_STAMP): poetry.lock
-	POETRY_VIRTUALENVS_IN_PROJECT=1 poetry install --only docs
+	POETRY_VIRTUALENVS_IN_PROJECT=1 poetry install --no-root --only docs
 	touch $(DOC_STAMP)
 
 docs: install-docs  ## Build documentation

```

Reply:
I like your proposal, so I did both 😎